### PR TITLE
Dat 2812 add support for image caption to portable infoboxes

### PR DESCRIPTION
**Pull request by RafLeszczynski**

### CODE REVIEW

File: extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeImage.php [hunk 1]
```diff
@@ -5,18 +5,22 @@
...
 		$ref = null;
 		$alt = $this->getValueWithDefault( $this->xmlNode->{self::ALT_TAG_NAME} );
+		$caption = $this->getValueWithDefault( $this->xmlNode->{self::CAPTION_TAG_NAME} );
 
 		wfRunHooks( 'PortableInfoboxNodeImage::getData', [ $title, &$ref, $alt ] );
 
 		return [
 			'url' => $this->resolveImageUrl( $title ),
-			'name' => ( $title ) ? $title->getDBkey() : '',
+			'name' => ( $title ) ?htmlspecialchars( $title->getText() ) : '',
```
Comment (idradm) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeImage.php]: Fix format please. Why do we need chars encoding here?

Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeImage.php]: :+1: - we consistently do escaping as close to rendering as possible

File: extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeImage.php [hunk 2]
```diff
@@ -5,18 +5,22 @@
...
+			'key' => ( $title ) ? htmlspecialchars( urlencode( $title->getDBKey() ) ) : '',
```
Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/services/Parser/Nodes/NodeImage.php]: And here. Per offline discussion, we should do the alterations just prior to passing this to the template

**Comment by SebastianMarzjan** [comment]
:+1:

**Pull request merged**